osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/GetOutagesRequestList.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.dto.valueobjects.smartmetering;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class GetOutagesRequestList implements Serializable {
```

Comment:
for my understanding, why is there a request list? and not a single request to get the outages from a meter?

Reply:
see comment above